code/datums/emergency_calls/emergency_call.dm

```diff
@@ -169,9 +169,9 @@
 					if(!istype(M)) //Something went horrifically wrong
 						candidates -= M
 						continue
-					if(M.current?.stat != DEAD)
-						candidates -= M //Strip them from the list, they aren't dead anymore.
-						continue
+					//if(M.current?.stat != DEAD)
```

Comment:
Why?

Reply:
Although definitely a shitty fix, if you got gibbed or ghosted you wouldn't have the dead flag and you couldn't be a part of the ERT, I can change this back if you want to.

Reply:
I'll have to either make a better check or fix setting flags